### PR TITLE
fix broken links and spelling

### PR DESCRIPTION
**Pull request by sedlund**

### CODE REVIEW

File: docs/network-setup.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ dhcp-boot=tag:ipxe,http://matchbox.example.com:8080/boot.ipxe
 log-queries
 log-dhcp
 
-# static DNS assignements
```
Comment (dghubble) [inline docs/network-setup.md:91]: assignments without the `'`

File: docs/deployment.md [hunk 2]
```diff
@@ -286,7 +286,7 @@ For large production environments, use a cache proxy or mirror suitable for your
 
 Review [network setup](https://github.com/poseidon/matchbox/blob/master/docs/network-setup.md) with your network administrator to set up DHCP, TFTP, and DNS services on your network. At a high level, your goals are to:
 
-* Chainload PXE firmwares to iPXE
```
Comment (dghubble) [inline docs/deployment.md:289]: ok as it was

**Review by dghubble — COMMENTED** [review]
thanks

**Comment by sedlund** [comment]
:+1:

**Pull request merged**